lib/rubocop/cop/metrics/block_length.rb

```diff
@@ -29,14 +29,16 @@ module Metrics
       #       content.
       #     HEREDOC
       #   end                 # 5 points
+      #
+      # NOTE: This cop does not apply for struct definitions.
```

Comment:
This is a nitpick. It may be clearer to users.

```suggestion
      # NOTE: This cop does not apply for `Struct` definitions.
```